src/Ilios/CoreBundle/Entity/Repository/CourseRepository.php

```diff
@@ -64,37 +66,6 @@ public function findBy(array $criteria, array $orderBy = null, $limit = null, $o
             $qb->setParameter(':programYears', $ids);
         }
 
-        if (array_key_exists('users', $criteria)) {
```

Comment:
Why take this out?  It isn't the best way to go, but would still be the only way to find courses for an arbitrary user.